3-enrich/emr-etl-runner/config/config.yml.sample

```diff
@@ -45,6 +45,7 @@ aws:
       master_instance_type: m1.medium
       core_instance_count: 2
       core_instance_type: m1.medium
+      core_instance_bid: 0.015
```

Comment:
The commit message should be the same as the issue title. In this case Mike used his own name for the PR and we have carried it over.